packages/flutter/lib/src/material/pickers/calendar_date_picker.dart

```diff
@@ -366,8 +366,7 @@ class _DatePickerModeToggleButtonState extends State<_DatePickerModeToggleButton
         children: <Widget>[
           Flexible(
             child: Semantics(
-              // TODO(darrenaustin): localize 'Select year'
-              label: 'Select year',
+              label: MaterialLocalizations.of(context).selectYearLabel,
```

Comment:
Might be worth naming/mentioning that this is a semantics label that will be used for VoiceOver and Talkback.
```suggestion
              label: MaterialLocalizations.of(context).selectYearSemanticsLabel,
```

Reply:
Renamed.